### ISSUE TITLE
disallow top 100 passwords

### ISSUE THREAD

**Issue by xLink** [Cysha/pxcms-auth#15]
mebe worth adding functionality in to kill these off?

**Comment by richard-clifford** [comment]
Yeah sounds good. I have a big list of common passwords - top 100, 1000, 1000000 etc

**Comment by NoelDavies** [comment]
Ability to import a list, add custom ones or custom patterns?
